webapp-django/crashstats/settings/local.py-dist

```diff
@@ -168,3 +168,19 @@ SECRET_KEY = 'you must change this'
 #           'component=General&groups=client-services-security',
 #    'label': 'File a bug in bugzilla'
 #}
+#
+# We're adding configman to django, and so we need to specify a path 
+# to the .ini files needed. The setting below is appropriate for local
+# development where you invoke manage.py in the webapp-django/ directory
+# this configuration is for our travis environment
+import os
+build_path = os.environ.get(
+    'TRAVIS_BUILD_DIR',
```

Comment:
We should perhaps move this to one of the tests. Don't we have something specific in the ./scripts/ files that is specific to travis?